code/player/TTTPlayer.Confirmation.cs

```diff
@@ -55,7 +55,7 @@ private void BecomePlayerCorpseOnServer(Vector3 force, int forceBone)
                 Rotation = Rotation
             };
 
-            corpse.KillerWeapon = LastDamageWeapon?.LibraryName;
+            // corpse.KillerWeapon = LastDamageWeapon?.LibraryName;
```

Comment:
So the waepon isn't passed anymore?

Reply:
Sorry, need to go back and this this, Good catch.